model/inline/src/main/scala/aqua/model/inline/raw/ApplyPropertiesRawInliner.scala

```diff
@@ -41,94 +74,103 @@ object ApplyPropertiesRawInliner extends RawInliner[ApplyPropertyRaw] {
     }
   }
 
-  private[inline] def removeProperty[S: Mangler: Exports: Arrows](
-    vm: ValueModel
-  ): State[S, (ValueModel, Inline)] =
-    vm match {
-      case VarModel(nameM, btm, propertyM) if propertyM.nonEmpty =>
-        for {
-          nameMM <- Mangler[S].findAndForbidName(nameM)
-        } yield VarModel(nameMM, vm.`type`, Chain.empty) -> Inline.preload(
-          // TODO use smth more resilient to make VarRaw from a flattened VarModel
-          nameMM -> ApplyPropertyRaw.fromChain(VarRaw(nameM, btm), propertyM.map(_.toRaw))
-        )
-      case _ =>
-        State.pure(vm -> Inline.empty)
+  private def flatVar[S: Mangler: Exports: Arrows](
```

Comment:
- Comment plz, what it does
- `flat` is not a verb, maybe `flatten`? Or `removeProperties`? Or `flattenProperties`?
- It looks like not all the typeclasses are needed here, I see only `Mangler` used